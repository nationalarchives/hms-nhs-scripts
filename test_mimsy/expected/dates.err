Warning: bad date format "March 21 1827". Should be like "Apr 01 1800".
Warning: bad date format "a bad date". Should be like "Apr 01 1800".
Warning: bad date format "03-21-1827". Should be like "Apr 01 1800".
Warning: bad date format "13-04-1827". Should be like "Apr 01 1800".
Warning: blank in "date of entry" at row 6 of input/dates.csv (volume 1, page 81)
Warning: blank in "date of discharge" at row 6 of input/dates.csv (volume 1, page 81)
