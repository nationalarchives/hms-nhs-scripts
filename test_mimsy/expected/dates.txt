DSH/1/81,admission number, 6925; date of entry, March 21 1827; name, John Reed; quality, Able Seaman; age, 32; place of birth, Shields; port sailed out of, ; years at sea, 01.5: 14; last services, Peggy; under what circumstances admitted (or nature of complaint), Rheumatism; date of discharge, a bad date; how disposed of, Shipped; number of days victualled, 20; admission number, 6926; date of entry, 03-21-1827; name, James Robinson; quality, Boy; age, 15; place of birth, Yarmouth; port sailed out of, ; years at sea, 00: 01.5; last services, Harby; under what circumstances admitted (or nature of complaint), Scald Head; date of discharge, 13-04-1827; how disposed of, To a/his Ship Cured; number of days victualled, 23; admission number, 6926; date of entry, 12-04-1900; name, James Robinson; quality, Boy; age, 15; place of birth, Yarmouth; port sailed out of, ; years at sea, 00: 01.5; last services, Harby; under what circumstances admitted (or nature of complaint), Scald Head; date of discharge, 12-04-1900; how disposed of, To a/his Ship Cured; number of days victualled, 23; admission number, 6926; date of entry, 12-04-1900; name, James Robinson; quality, Boy; age, 15; place of birth, Yarmouth; port sailed out of, ; years at sea, 00: 01.5; last services, Harby; under what circumstances admitted (or nature of complaint), Scald Head; date of discharge, 12-04-1900; how disposed of, To a/his Ship Cured; number of days victualled, 23; admission number, 6925; date of entry, ; name, John Reed; quality, Able Seaman; age, 32; place of birth, Shields; port sailed out of, ; years at sea, 01.5: 14; last services, Peggy; under what circumstances admitted (or nature of complaint), Rheumatism; date of discharge, ; how disposed of, Shipped; number of days victualled, 20;
